app/views/best.html

```diff
@@ -40,7 +40,7 @@
               <span ng-if="item.normalStats[$index].scaled > 0"><small ng-bind="item.normalStats[$index].scaled"></small>/<small ng-bind="item.normalStats[$index].split"></small></span>
             </div>
           </div>
-          <div dim-store-item item-data="item" store-data="normal.getStore(item.owner)"></div>
+          <div dim-simple-item item-data="item"></div>
```

Comment:
could we possibly change this one back? i've liked being able to see and move items on the quality on the armor comparison view. was it hurting perf that much?